steps/operators_test.py

```diff
@@ -45,3 +49,31 @@ def test_concatenate_qubit_operator_lists():
 
     assert group_read == h2_hamiltonian_grouped
 
+def test_get_one_qubit_hydrogen_hamiltonian():
+
+    # Interaction Hamiltonian of H2 (0.74 A, STO-3G) generated using openfermion-psi4
+    constant = 0.7151043387432434
+    # Load one body tensor from .npz file
+    file_handle_1b = np.load('h2_one_body_tensor.npz')
+    one_body_tensor = file_handle_1b.get('arr_0')
+    # Load one body tensor from .npz file
+    file_handle_2b = np.load('h2_two_body_tensor.npz')
+    two_body_tensor = file_handle_2b.get('arr_0')
+    # Assemble interaction operator
+    h2_hamiltonian_int = InteractionOperator(constant=constant, one_body_tensor=one_body_tensor, two_body_tensor=two_body_tensor)
+    save_interaction_operator(h2_hamiltonian_int, 'interaction-operator.json')
+
+    get_one_qubit_hydrogen_hamiltonian('interaction-operator.json')
+    h2_1qubit = load_qubit_operator('qubit-operator.json')
+    h2_1qubit_sparse = get_sparse_operator(h2_1qubit, n_qubits=1)
+    h2_1qubit_dense = h2_1qubit_sparse.toarray()
+    print(h2_1qubit_dense)
+
+    e_1q = eigvalsh(h2_1qubit_dense)
+
+    gs_4q = get_ground_state(get_sparse_operator(h2_hamiltonian_int))
+
+    assert isclose(e_1q[0], gs_4q[0])
+
+    os.remove('interaction-operator.json')
+    os.remove('qubit-operator.json')
```

Comment:
I think we should remove files before the `assert`, to make sure we clean up before the test fails, for example.

Reply:
Good point! Will do!